go.mod

```diff
@@ -3,7 +3,12 @@ module knative.dev/client-pkg
 go 1.16
 
 require (
+	github.com/mitchellh/go-homedir v1.1.0
+	github.com/spf13/cobra v1.2.1
 	gotest.tools/v3 v3.0.3
-	k8s.io/apimachinery v0.21.4
+	k8s.io/api v0.20.7
+	k8s.io/apimachinery v0.20.7
 	knative.dev/hack v0.0.0-20210806075220-815cd312d65c
+	knative.dev/pkg v0.0.0-20210803160015-21eb4c167cc5
```

Comment:
Doesn't depending on these kind of defeat the purpose of having a `client-pkg` repo in the first place? :thinking: Wasn't the new repo supposed to be easier to vendor than straight `pkg`?

Reply:
Rereading https://github.com/knative/community/issues/523 I might've been plain wrong. Feel free to ignore.

Reply:
The main purpose of `knative/client-pkg` is to extract shared code that can be used by `kn` and its plugins, to avoid cyclic dependencies. It is also allowed to have dependencies on serving & eventing, something straight `knative/pkg` can not do.